lintro/utils/tool_executor.py

```
@@ -121,11 +126,207 @@ def _run_fix_with_retry(
+            initial_issues=first_pass_initial_issues,
         )
 
     return result
 
 
+def _warn_ai_fix_disabled(
+    *,
+    action: Action,
+    ai_fix: bool,
+    ai_enabled: bool,
+    logger: Any,
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `logger`

(ANN401)